letsencrypt/configuration.py

```diff
@@ -18,8 +18,7 @@ class NamespaceConfig(object):
     paths defined in :py:mod:`letsencrypt.constants`:
 
       - `accounts_dir`
-      - `cert_dir`
-      - `cert_key_backup`
```

Comment:
this shouldn't have been removed (at least not in this PR), as other instances are floating around